rules-reviewed/jboss-eap/websphere/websphere.windup.xml

```diff
@@ -0,0 +1,712 @@
+<?xml version="1.0"?>
+<ruleset xmlns="http://windup.jboss.org/v1/xml" id="websphere">
+    <metadata>
+      <description>
+         This ruleset provides analysis of WebSphere proprietary classes and constructs that may require individual attention when migrating to JBoss EAP 6+.
+      </description>
+      <dependencies>
+         <addon id="org.jboss.windup.rules,windup-rules-javaee,2.2.0.Final"/>			
+         <addon id="org.jboss.windup.rules,windup-rules-java,2.2.0.Final"/>
+      </dependencies>
+      <sourceTechnology id="websphere" versionRange="(7,)"/>
+      <targetTechnology id="eap" versionRange="[6,)"/>
+   </metadata>
+    <rules>
+        <rule id="websphere-7Apvvd">
```

Comment:
I am also concerned about the IDs. You follow the structure like websphere-01000. However since my rules are starting somewhere in the middle, I need to wait for the first half to be reviewed to know with which number to start